src/app/Page.tsx

```diff
@@ -6,6 +6,8 @@ import Unlock from "app/pages/Unlock";
 import Welcome from "app/pages/Welcome";
 import ImportWallet from "app/pages/ImportWallet";
 import CreateWallet from "app/pages/CreateWallet";
+
```

Comment:
Pls, remove useless line break.

And, for the future, in this project, ALL import declarations must be used without line breaks and with ordering: NPM, `lib/*` dir, `app/*` dir, `local`(from "./Kek.module.css").
Thanks 🙏